dev-config.js

```diff
@@ -9,6 +25,10 @@ module.exports = function createDevConfig(serviceWorker, favPath) {
 		plugins.push("new FaviconsWebpackPlugin('" + favPath + "')");
 	}
 
+	if (manifestDetails) {
+		plugins.push("new CopyWebpackPlugin([{ from: '" + getManifestPath(manifestDetails) + "', to: '/build'}])");
```

Comment:
the build should be in the question right. We cannot assume the output dir, so it is better to get this information from users.